app/controllers/application_controller.rb

```diff
@@ -1,3 +1,10 @@
 class ApplicationController < ActionController::Base
   protect_from_forgery with: :exception
+
+  def current_user
+    begin
+      @current_user ||= User.find(session[:user_id]) if session[:user_id]
+    rescue
```

Comment:
Is this necessary? Do we need some kind of error message to redirect?

Reply:
this is necessary to get the current user if one is not already logged in

Reply:
Awesome, thanks!